src/musikcube/app/window/TransportWindow.cpp

```diff
@@ -304,6 +304,13 @@ static size_t writePlayingFormat(
             }
         }
 
+        /* any % in the value string might be parsed by wprintw, so replace it */
+        std::size_t perc_index = value.find("%");
```

Comment:
Would you mind rename `perc_index` to `percentIndex` or `percentSignIndex` to more closely match our coding conventions? :)

Reply:
oh, typed too much Rust lately, will do :)